readme.md

```diff
@@ -56,13 +56,19 @@ banner-HD.jpg 2x, banner-phone.jpg 100w, banner-super-HD.jpg 3x
 
 ## API
 
-### .parse()
+### .parse(string, [options])
 
 Accepts a srcset string and returns an array of objects with the possible properties: `url` (always), `width`, `density`.
 
-### .stringify()
+If options is set to `{strict: false}`, it will attempt to parse invalid input. Otherwise errors are thrown on invalid input.
 
-Accepts an array of objects with the possible properties: `url` (required), `width`, `density` and returns a srcset string.
+### .stringify(array, [options])
+
+Accepts an array of objects with the possible properties: `url` (required), and one of `width` or `density` and returns a srcset string.
+
+By default it will validate the objects and throw an error if they would produce an invalid srcset string.
+
+If options is set to `{strict: false}`, it will skip validation.
```

Comment:
You need to update index.d.ts too. Readme and index.d.ts should be in sync.

https://github.com/sindresorhus/typescript-definition-style-guide/tree/57a925bb5b3eb46a4666eab31e1c883a26b7bfb5